table/tables/tables.go

```diff
@@ -723,6 +753,49 @@ func GetColDefaultValue(ctx context.Context, col *model.ColumnInfo) (interface{}
 	return col.DefaultValue, true, nil
 }
 
+// EncodeRecordKey encodes the string value to a byte slice.
+func EncodeRecordKey(tablePrefix string, h int64, columnID int64) []byte {
```

Comment:
EncodeRecordKey and DecodeRecordKey is not matched, we must handle TablePrefix outer manually for encode but automatically for decode. 

I think we can handle TablePrefix both innner in encode and decode, because for record, the first byte is "t" certainly.